abstract.tex

```diff
@@ -31,17 +31,35 @@
+%
+This abstract presents the outline for the code-to-code verification of Moltres.
+Moltres \cite{lindsay_introduction_2018} is an open source coupled neutronics/thermal hydraulics simulation tool for simulating molten salt reactors (MSR).
+MSRs differ greatly from conventional solid-fuelled reactors, particularly in their neutronics and thermal-hydraulics behaviours.
+They present unique computational challenges that many existing computational tools fail to address effectively.
+Strong coupling between neutronics and thermal-hydraulics modeling is an important requirement for new MSR simulation tools due to the strong temperature reactivity coefficients arising from Doppler effects and changes in the fuel salt density. 
 
-Moltres is an open source coupled neutronics/thermal hydraulics simulation tool for simulating MSRs \cite{lindsay_introduction_2018}. It is under active development at the University of Illinois. It is an application code built using the MOOSE finite element framework \cite{gaston_moose:_2009}. Moltres is developed as a solver for the coupled multi-group neutron diffusion, temperature and delayed neutron precursor governing equations. As a multi-group neutronics solver on a coarse mesh, Moltres imposes relatively low computational loads compared to continuous energy neutronics solvers and computational fluid dynamics thermal-hydraulic solvers employed by other researchers in the study of MSRs. This allows for steady-state, transient and accident analyses with relatively low computational requirements.
+The development of novel simulation tools for MSRs is an essential step towards understanding the behaviour of MSRs under various conditions, one of which is the changing fuel compositions across the lifespan of MSRs.
+The fuel composition in an MSR continuously evolves over time as online refueling replenishes fissile material while removing fission product poisons. The impact of changing fuel composition on the safety parameters such as the reactivity feedback coefficient must be studied extensively.
 
-This abstract presents the outline for the code-to-code verification of Moltres. The reactor model used for this verification is the Molten Salt Fast Reactor. The MSFR is a reference design for a fast-spectrum molten salt reactor developed under the EVOL and SAMOFAR projects\cite{serp_molten_2014}. There is abundant MSFR parameter and simulation data available from various authors \cite{fiorina_modelling_2014} \cite{pettersen_coupled_2016} for the reference model to perform the code-to-code verification.
+Moltres is an application code built in the MOOSE finite element framework \cite{gaston_moose:_2009}.
```

Comment:
- unnecessary passive voice.
- don't use the word "code" in this way.  (application is a noun... there's no reson to add the word code... and "software package" or "computational tool" or similar is preferable to "code" )